src/OrchardCore/OrchardCore.Lucene.Core/QueryProviders/SimpleQueryStringQueryProvider.cs

```diff
@@ -14,9 +15,19 @@ public Query CreateQuery(ILuceneQueryService builder, LuceneQueryContext context
             }
 
             var queryString = query["query"]?.Value<string>();
-            var fields = query["fields"]?.Value<string>() ?? "";
-            var defaultOperator = query["default_operator"]?.Value<string>() ?? "and";
-            var queryParser = new SimpleQueryParser(context.DefaultAnalyzer, "");
+            var fields = query["fields"]?.Values<string>() ?? new string[0];
+            var defaultOperator = query["default_operator"]?.Value<string>().ToLowerInvariant() ?? "or";
```

Comment:
I would prefer not to change the default. I assume that the more terms you add, the less results you get. Unless the Elastic search states the opposite.